pyk4a/pyk4a.cpp

```diff
@@ -303,6 +303,112 @@ extern "C" {
         }
     }
 
+    static PyObject* calibration_3d_to_3d(PyObject* self, PyObject *args){
+        k4a_result_t res;
+        k4a_calibration_t calibration;
+        k4a_float3_t source_point3d_mm;
+        k4a_float3_t target_point3d_mm;
+        k4a_calibration_type_t source_camera;
+        k4a_calibration_type_t target_camera;
+        int source_point_x;
+        int source_point_y;
+        int source_point_z;
+        k4a_device_configuration_t config = K4A_DEVICE_CONFIG_INIT_DISABLE_ALL;
+        
+        PyArg_ParseTuple(args, "IIIIIIIIIIIIII", 
+                &source_point_x,
+                &source_point_y,
+                &source_point_z,
+                &source_camera,
+                &target_camera,
+                &config.color_format,
+                &config.color_resolution,
+                &config.depth_mode,
+                &config.camera_fps,
+                &config.synchronized_images_only,
+                &config.depth_delay_off_color_usec,
+                &config.wired_sync_mode,
+                &config.subordinate_delay_off_master_usec,
+                &config.disable_streaming_indicator);
+        
+        source_point3d_mm.xyz.x = source_point_x;
+        source_point3d_mm.xyz.y = source_point_y;
+        source_point3d_mm.xyz.z = source_point_z;
+
+        k4a_result_t result;
+        result = k4a_device_get_calibration(device, config.depth_mode,
```

Comment:
I would prefer if we could use the calibration without a device and could set the calibration with a json. The method `calibration_set_from_raw` already load the json but only `k4a_transformation_t` is exposed. We could expose the `k4a_calibration_t` the same way (or find a better way without global variable...).

Reply:
global variables are not ideal, but short of passing the data between python and C, do you have ideas? We could just pass the depth and color_resolution... it's just 2 integers. Global variable aren't pretty but given this is more C than CPP, there isn't an object that could have a property set... but if we have it for transformation, why not for calibration? Thoughts?